TypeCobol/Compiler/Symbols/Builtins.cs

```diff
@@ -58,18 +59,22 @@ static Builtins()
 
             Boolean = new TypedefSymbol(string.Intern("Bool"));
             Boolean.Type = new TypedefType(Boolean, new Type(Type.Tags.Boolean));
+            Boolean.Type.SetFlag(Symbol.Flags.Strong, true, false);
```

Comment:
For me these types (Bool, Date, Currency and String) are pure TypeCobol types and therefore are Strict.

Reply:
They are marked as Strong in their TypeDefinition counterparts, that's why we applied the Strong flag here.

Reply:
I agree with @fm-117

Reply:
Ok